test/raml-files/markdown/SOME_DOC.md

```diff
@@ -0,0 +1,9 @@
+# RAML Parser Doc
+[![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/raml-org/raml-js-parser?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
If this document is just added for testing purposes, I would think it less confusing and safer to use a generic lorem-ipsum type without external references or text that may be considered real documentation.